app/furniture/marketplace/cart_product/notes/new.html.erb

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
unnecessary blank line

Reply:
These are the most necessary blank lines!